MatrixSDK/Room/Polls/PollModels.swift

```diff
@@ -0,0 +1,49 @@
+// 
+// Copyright 2021 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+class Poll {
+    enum Kind {
+        case disclosed
+        case undisclosed
+    }
+    
+    class AnswerOption {
+        var id: String!
+        var text: String!
```

Comment:
Here the properties are set with initializer we should not use unwrap optional or var
```suggestion
        let id: String
        let text: String
```

Reply:
Same as above ⏫